sdcm/cluster.py

```diff
@@ -5573,10 +5572,9 @@ def reconfigure_scylla_monitoring(self):
                 python3 genconfig.py -s -n -d {self.monitoring_conf_dir} {monitoring_targets}
             """), verbose=True)
 
-            if cluster_backend != "docker":
-                node.remoter.sudo(f"docker run --rm -v {self.monitoring_conf_dir}:/workdir"
-                                  f" mikefarah/yq:3 yq w -i node_exporter_servers.yml"
-                                  f" '[0].targets[+]' ''[{node.private_ip_address}]:9100''", verbose=True)
+            with node._remote_yaml(f'{self.monitoring_conf_dir}/node_exporter_servers.yml') as exporter_yaml:  # pylint: disable=protected-access
```

Comment:
did you test it on docker backend?

Reply:
doing the docker provision test now

Reply:
but it's only for case of adding new node, and so it should be tested directly

Reply:
`reconfigure_scylla_monitoring` doesn't take node param, it reconfigures everything.

Reply:
There is a loop on the nodes

Reply:
monitoring nodes

Reply:
monitoring nodes

Reply:
It's adding the node exporter of the monitor

Reply:
no, but we are using that code multiple times on scylla yaml, should work as well.